### ISSUE TITLE
Changes in license check tool output

### ISSUE THREAD

**Issue by ffhreijnen** [eclipse/dash-licenses#126]
For Eclipse ESCET we integrated the license check into a daily Jenkins job. The output changed today. While that happens regularly, now some dependencies are suddenly restricted.

Did anything change?

```
00:59:55  Checking for differences between generated and stored dependency lists...
00:59:55  --- DEPENDENCIES.txt	2021-12-16 23:58:33.018212470 +0000
00:59:55  +++ DEPENDENCIES.generated.processed.txt	2021-12-16 23:59:55.330867075 +0000
00:59:55  @@ -5,7 +5,7 @@
00:59:55   maven/mavencentral/org.codehaus.groovy/groovy-ant/3.0.8, Apache-2.0, approved, clearlydefined
00:59:55   maven/mavencentral/org.codehaus.groovy/groovy-groovydoc/3.0.8, Apache-2.0, approved, #819
00:59:55   maven/mavencentral/org.codehaus.groovy/groovy-json/3.0.8, Apache-2.0, approved, clearlydefined
00:59:55  -maven/mavencentral/org.codehaus.groovy/groovy/3.0.8, Apache-2.0, approved, #818
00:59:55  +maven/mavencentral/org.codehaus.groovy/groovy/3.0.8, Apache-2.0, restricted, clearlydefined
00:59:55   p2/orbit/p2.eclipse-plugin/com.google.guava/30.1.0.v20210127-2300, Apache-2.0 AND LicenseRef-Public-Domain, approved, CQ23002
00:59:55   p2/orbit/p2.eclipse-plugin/com.ibm.icu/67.1.0.v20200706-1749, ICU AND Unicode-TOU AND BSD-3-Clause AND BSD-2-Clause AND LicenseRef-ipadic-license AND LicenseRef-Public-Domain, approved, CQ22320
00:59:55   p2/orbit/p2.eclipse-plugin/com.sun.jna.platform/5.8.0.v20210406-1004, Apache-2.0 OR LGPL-2.1-or-later, approved, CQ23218
00:59:55  @@ -13,22 +13,22 @@
00:59:55   p2/orbit/p2.eclipse-plugin/io.github.java-diff-utils/4.8.0.v20201027-1614, Apache-2.0, approved, CQ22697
00:59:55   p2/orbit/p2.eclipse-plugin/javax.annotation/1.3.5.v20200909-1856, EPL-1.0, approved, orbit
00:59:55   p2/orbit/p2.eclipse-plugin/javax.inject/1.0.0.v20091030, Apache-2.0, approved, CQ3555
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.anim/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.anim/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55   p2/orbit/p2.eclipse-plugin/org.apache.batik.awt.util/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.bridge/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.constants/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.css/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.dom.svg/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.dom/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.ext/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.gvt/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.i18n/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.parser/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.script/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.util/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.batik.xml/1.14.0.v20210324-0332, Apache-2.0, approved, #196
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.bridge/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.constants/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.css/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.dom.svg/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.dom/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.ext/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.gvt/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.i18n/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.parser/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.script/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.util/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.batik.xml/1.14.0.v20210324-0332, Apache-2.0, approved, orbit
00:59:55   p2/orbit/p2.eclipse-plugin/org.apache.commons.exec/1.1.0.v201301240602, Apache-2.0, approved, CQ6553
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.commons.io/2.8.0.v20210415-0900, Apache-2.0, approved, CQ23182
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.commons.io/2.8.0.v20210415-0900, Apache-2.0, approved, orbit
00:59:55   p2/orbit/p2.eclipse-plugin/org.apache.commons.jxpath/1.3.0.v200911051830, Apache-2.0, approved, CQ10790
00:59:55   p2/orbit/p2.eclipse-plugin/org.apache.commons.lang/2.6.0.v201404270220, Apache-2.0, approved, CQ6183
00:59:55   p2/orbit/p2.eclipse-plugin/org.apache.commons.lang3/3.1.0.v201403281430, Apache-2.0, approved, CQ5902
00:59:55  @@ -38,9 +38,9 @@
00:59:55   p2/orbit/p2.eclipse-plugin/org.apache.xalan/2.7.2.v20201124-1837, Apache-2.0, approved, CQ9587
00:59:55   p2/orbit/p2.eclipse-plugin/org.apache.xerces/2.12.1.v20210115-0812, Apache-2.0 AND W3C-19980720, approved, CQ16951
00:59:55   p2/orbit/p2.eclipse-plugin/org.apache.xml.resolver/1.2.0.v201005080400, Apache-2.0, approved, CQ1441
00:59:55  -p2/orbit/p2.eclipse-plugin/org.apache.xmlgraphics/2.6.0.v20210409-0748, Apache-2.0, approved, CQ23228
00:59:55  -p2/orbit/p2.eclipse-plugin/org.bouncycastle.bcpg/1.65.0.v20200527-1955, Apache-2.0, approved, CQ21975
00:59:55  -p2/orbit/p2.eclipse-plugin/org.bouncycastle.bcprov/1.65.1.v20200529-1514, MIT AND LicenseRef-Public-Domain, approved, CQ21977
00:59:55  +p2/orbit/p2.eclipse-plugin/org.apache.xmlgraphics/2.6.0.v20210409-0748, unknown, restricted, none
00:59:55  +p2/orbit/p2.eclipse-plugin/org.bouncycastle.bcpg/1.65.0.v20200527-1955, unknown, restricted, none
00:59:55  +p2/orbit/p2.eclipse-plugin/org.bouncycastle.bcprov/1.65.1.v20200529-1514, unknown, restricted, none
00:59:55   p2/orbit/p2.eclipse-plugin/org.eclipse.ant.core/3.6.0.v20210415-1314, EPL-2.0, approved, eclipse.platform
00:59:55   p2/orbit/p2.eclipse-plugin/org.eclipse.compare.core/3.6.1000.v20201020-1107, EPL-2.0, approved, eclipse.platform
00:59:55   p2/orbit/p2.eclipse-plugin/org.eclipse.compare/3.8.0.v20210415-0713, EPL-2.0, approved, eclipse.platform
````

And for the restricted dependencies check:
```
00:59:55  Checking for restricted dependencies...
00:59:55  maven/mavencentral/org.codehaus.groovy/groovy/3.0.8, Apache-2.0, restricted, clearlydefined
00:59:55  p2/orbit/p2.eclipse-plugin/org.apache.xmlgraphics/2.6.0.v20210409-0748, unknown, restricted, none
00:59:55  p2/orbit/p2.eclipse-plugin/org.bouncycastle.bcpg/1.65.0.v20200527-1955, unknown, restricted, none
00:59:55  p2/orbit/p2.eclipse-plugin/org.bouncycastle.bcprov/1.65.1.v20200529-1514, unknown, restricted, none
```

**Comment by waynebeaton** [comment]
It looks like this one was PEBKAC: I messed up one of the files that maps between content Ids and status while applying an update. I've restored the file and ran the tool against the Eclipse ESCET repository. It appears that you're back in business.

I'll mitigate against this happening again by "locking" the existing mapping files and creating new ones for future additions.

I can explain any part of what I've said above if anybody cares.
